website_v2/gcloud-powershell-website-v2/www/static/app/mainController.js

```diff
@@ -0,0 +1,12 @@
+var app = angular.module('powershellSite');
+
+app.controller('MainController',
+    function($scope, $route, $routeParams, $location, $http) {
+    $scope.$route = $route;
+    $scope.$location = $location;
+    $scope.$routeParams = $routeParams;
+    $http.get('static/_data/cmdletsFull.json')
+       .then(function(res) {
```

Comment:
What happens if this fails?  It shouldn't because this is a static file but an error handler would be nice